lib/ramble/ramble/schema/types.py

```diff
@@ -8,12 +8,31 @@
 
 """Base types for building schema files from"""
 
+from enum import Enum
 
-class OUTPUT_CAPTURE:
-    STDERR = "2>"
-    STDOUT = ">>"
-    ALL = "&>"
-    DEFAULT = STDOUT
+
+class OUTPUT_CAPTURE(Enum):
```

Comment:
I know this came from `schema.types` but would it still work if this was moved into `ramble.util` ?

Reply:
I can't see why not, let me try

Reply:
Done